webapp/src/components/data_selectors/jira_epic_selector/jira_epic_selector.test.tsx

```diff
@@ -28,7 +28,7 @@ describe('components/JiraEpicSelector', () => {
     test('should match snapshot', () => {
         const props = {...baseProps};
         const wrapper = shallow<JiraEpicSelector>(
-            <JiraEpicSelector {...props}/>
+            <JiraEpicSelector {...props}/>,
```

Comment:
Shouldn't the test fail here since `theme` is a required prop? And based on the snapshot it's no longer being provided